internal/envconfig/envconfig.go

```diff
@@ -4,13 +4,14 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//	http://www.apache.org/licenses/LICENSE-2.0
+//     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
It seems that was a few places with different indentation on this line, applied one for all files.

Reply:
@ckadner ^

Reply:
https://github.com/kserve/modelmesh-runtime-adapter/pull/68#discussion_r1408282261

Reply:
To be addressed in #69